cfme/tests/configure/test_rest_access_control.py

```diff
@@ -819,3 +820,103 @@ def test_delete_users_from_collection(self, users):
             initialEstimate: 1/4h
         """
         delete_resources_from_collection(users)
+
+
+COMMON_FEATURES = [
+    "Services",
+    "Compute",
+    "Configuration",
+    "Networks",
+    "Storage",
+    "Control",
+    "Automation",
+    "Monitor",
+    "Help",
+    "API",
+    "Service UI",
+]
+
+FEATURES_511 = COMMON_FEATURES + [
+    "Overview",
+    "Migration",
+    "User Settings",
+    "All VM and Instance Access Rules",
+    "Main Configuration",
+]
+
+FEATURES_510 = COMMON_FEATURES + [
+    "Cloud Intel",
+    "Red Hat Insights",
+    "Optimize",
+    "Access Rules for all Virtual Machines",
+]
+
+
+@pytest.fixture(params=[True, False], ids=["individual", "everything"])
+def create_role(appliance, request):
+    role_name = "API-role-{}".format(fauxfactory.gen_alpha(4))
+    features = FEATURES_511 if appliance.version > "5.11" else FEATURES_510
+
+    view = navigate_to(appliance.collections.roles, "Add")
```

Comment:
Q: why can't you use the `appliance.collections.role.create()` method here? Is it because of the need to `check_node` below? 

If so, can this be a feature enhancement of the `create()` method? Or is it really specific to this test?

Reply:
Thanks for pointing this out John! I think I can use the normal create method for this test. I'm working on it!